packages/core/src/common/util/migration/schemas/15.json

```diff
@@ -28,13 +28,17 @@
             "dataFormatVersion": {
               "type": "string",
               "const": "15"
+            },
```

Comment:
Why is version also added to 15.json and not only 16?

Reply:
Good catch - that is an error and will be fixed before merging